CHANGELOG.md

```diff
@@ -20,11 +20,15 @@
   [bootstraponline](https://github.com/bootstraponline)
   [#689](https://github.com/realm/SwiftLint/issues/689)
 
+* Made `- returns:` doc optional for constructors.
```

Comment:
missing two trailing spaces as indicated in https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

I'd also prefer if you replaced the word "constructors" with "initializers" because constructors also include the family of static functions that return `Self` and this doesn't apply to that.